Sources/Extensions/UIKit/UIImageExtensions.swift

```diff
@@ -74,7 +74,7 @@ public extension UIImage {
     public func scaled(toHeight: CGFloat, opaque: Bool = false, with orientation: UIImageOrientation? = nil) -> UIImage? {
```

Comment:
While we're at it fixing these functions, can you remove the `orientation` param for both of them? It's unused.

Reply:
Done.